regression-tests/18-compile-arm-ports/Makefile

```diff
@@ -54,6 +54,8 @@ sensniff/ev-aducrf101mkxz \
 cfs-coffee/cc2538dk \
 cfs-coffee/openmote-cc2538 \
 cfs-coffee/zoul \
+fat/zoul:BOARD=remote-reva \
```

Comment:
It has a bug in the design, so even if some features of the Micro-SD are working, the SD card detection is not working.  We are not manufacturing `remote-reva` anymore, but I'm OK to leave the code as it is for the `remote-reva`, in case anyone is willing to try it out